test/unit/modules/sfc/sfc-parser.spec.js

```diff
@@ -99,7 +102,7 @@ describe('Single File Component parser', () => {
       }
       </test>
     `)
-    expect(res.customBlocks.length).toBe(3)
```

Comment:
This is actually the important part of the test 😆 
As said below, I think we should just add another test